src/js/extractors/factories/callExpression.ts

```diff
@@ -31,6 +50,11 @@ export function callExpressionExtractor(calleeName: string | string[], options:
             if (matches) {
                 let message = extractArguments(callExpression, options.arguments);
                 if (message) {
+                    message.text = normalizeContent(message.text, contentOptions);
+                    message.textPlural = message.textPlural ?
+                        normalizeContent(message.textPlural, contentOptions) : message.textPlural;
+                    message.context = message.context ?
+                        normalizeContent(message.context, contentOptions) : message.context;
```

Comment:
That make sense, but `contentOptions` are not available in the `extractArguments` function. So we need to add an additional argument `IContentOptions` or use the whole `IJsExtractorOptions` object.

Reply:
True, I think we can just pass along the whole options object.

Reply:
Passing the whole options object requires a casting in `extractArguments` because the `content` property is defined as Partial.
```typescript
let contentOptions = <IContentOptions>options.content;
```
That seems a bit strange to me, so I decide to add a additional `IContentOptions` argument.

But if this casting is ok for you, I can change this.

Reply:
Good call 👍